cinema/views.py

```diff
@@ -127,6 +123,28 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="actors",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by actor id"
+            ),
+            OpenApiParameter(
+                name="genres",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by genre id"
+            ),
+            OpenApiParameter(
+                name="title",
+                type=str,
+                description="Filter by title"
+            )
+        ]
+    )
```

Comment:
I don't think it's a big issue, it would probably be good to have usage examples in the descriptions.